scripts/pytest.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "Running Unit tests"
+
+pytest --random-order --cov=freqtrade --cov-config=.coveragerc freqtrade/tests/
```

Comment:
the path for tests changed from `freqtrade/tests/` to `tests/` in the latest develop versions.

i have a split opinion about this file overall

We can move this command as `command` to the `docker-compose.develop.yml` since it's only one command, or it should include all ci commands (`flake8`, `mypy freqtrade/ scripts/`).